Exercise-13/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Friends</title>
+
+    <!-- external font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;500&display=swap" rel="stylesheet" />
+
+    <!-- external css -->
+    <link rel="stylesheet" href="assets/css/styles.css" />
+  </head>
+  <body>
+    <!-- header section start -->
+    <header class="header-content">
+      <span>FRIENDS</span>
```

Comment:
could you let me know if this could be heading tag instead of span ?

Reply:
Yes, I have seen that it is best to use heading tags for best seo practices. I have changed it to h1 tag